etc/prod/docker-compose.windows.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.7'
+
+services:
+
+  php:
+    environment:
+      XDEBUG_CONFIG: remote_connect_back=0 remote_host=host.docker.internal remote_port=9000
```

Comment:
Xdebug in production?

Reply:
It says windows in name file.

Reply:
Why not xdebug? This is about run the application in production mode, not a deploy process.